# HG changeset patch
# User oleh-derevenko
# Date 1547341015 -7200
# Node ID 96154dbbff9437c721130272f6f20584b5ec96d8
# Parent  7755c9d58f8bb0eaf2592a49c209d4d9817e3f8e
Fixed: String formatting arguments/types fixed to match in the libccd tests

diff --git a/libccd/src/testsuites/bench.c b/libccd/src/testsuites/bench.c
--- a/libccd/src/testsuites/bench.c
+++ b/libccd/src/testsuites/bench.c
@@ -246,7 +246,7 @@
         cycles = atol(argv[1]);
     }
 
-    fprintf(stdout, "Cycles: %u\n", cycles);
+    fprintf(stdout, "Cycles: %zu\n", cycles);
     fprintf(stdout, "\n");
 
     boxbox();
diff --git a/libccd/src/testsuites/bench2.c b/libccd/src/testsuites/bench2.c
--- a/libccd/src/testsuites/bench2.c
+++ b/libccd/src/testsuites/bench2.c
@@ -252,7 +252,7 @@
         cycles = atol(argv[1]);
     }
 
-    fprintf(stdout, "Cycles: %u\n", cycles);
+    fprintf(stdout, "Cycles: %zu\n", cycles);
     fprintf(stdout, "\n");
 
     boxbox();
diff --git a/libccd/src/testsuites/common.c b/libccd/src/testsuites/common.c
--- a/libccd/src/testsuites/common.c
+++ b/libccd/src/testsuites/common.c
@@ -9,7 +9,7 @@
     ccd_quat_t rot;
     ccd_vec3_t axis, vpos, vpos2;
     ccd_real_t angle, x, y;
-    size_t i;
+    unsigned int i;
 
     ccdVec3Set(&axis, 0., 0., 1.);
     ccdVec3Set(&vpos, 0., c->radius, 0.);
diff --git a/libccd/src/testsuites/polytope.c b/libccd/src/testsuites/polytope.c
--- a/libccd/src/testsuites/polytope.c
+++ b/libccd/src/testsuites/polytope.c
@@ -101,7 +101,7 @@
     ccd_pt_face_t *f[4];
     ccd_vec3_t u;
     int res;
-    size_t i;
+    unsigned int i;
 
     DBG2("------");
 
